tests/core/fixtures.py

```diff
@@ -112,13 +123,52 @@ def chain(chaindb):
 
 
 @pytest.fixture
-def shard_chain():
-    shard_chaindb = BaseChainDB(get_db_backend(), account_state_class=ShardingAccountStateDB)
-    return chain(shard_chaindb)
+def shard_chain(shard_chaindb):
+    shard_chaindb = shard_chaindb
+    genesis_params = {
+        "bloom": 0,
+        "coinbase": to_canonical_address("8888f1f195afa192cfee860698584c030f4c9db1"),
+        "difficulty": 131072,
+        "extra_data": b"B",
+        "gas_limit": 3141592,
+        "gas_used": 0,
+        "mix_hash": decode_hex(
+            "56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421"),
+        "nonce": decode_hex("0102030405060708"),
+        "block_number": 0,
+        "parent_hash": decode_hex(
+            "0000000000000000000000000000000000000000000000000000000000000000"),
+        "transaction_root": constants.EMPTY_SHA3,
+        "receipt_root": constants.EMPTY_SHA3,
+        "timestamp": 1422494849,
+        "uncles_hash": decode_hex(
+            "1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347")
+    }
+    funded_addr = to_canonical_address("a94f5374fce5edbc8e2a8697c15331677e6ebf0b")
+    initial_balance = 10000000000
+    genesis_state = {
+        funded_addr: {
+            "balance": initial_balance,
+            "nonce": 0,
+            "code": b"",
+            "storage": {}
+        }
+    }
+    klass = Shard.configure(
+        name='TestChain',
+        vm_configuration=(
+            (constants.GENESIS_BLOCK_NUMBER, ShardingVM),
+        ))
+    shard = klass.from_genesis(shard_chaindb, genesis_params, genesis_state)
+    shard.funded_address = funded_addr
```

Comment:
I'd advocate for making two new fixtures `funded_address` and `funded_address_initial_balance`.  Any test which needs access to the address can then just use the fixture.  Seems cleaner than assigning them as a property to the `shard` instance.